src/components/themed/ThemedButtons.js

```diff
@@ -150,6 +150,15 @@ export function RightChevronButton(props: { text: string, onPress: () => void, p
   )
 }
 
+export function IconButton(props: Props) {
```

Comment:
This component is redundant, you already have the 'ButtonBox' component is the same file which is exactly the same.